packages/design-system/src/components/command.tsx

```diff
@@ -30,23 +30,23 @@ import { Button } from "./button";
 import { Popover, PopoverContent, PopoverTrigger } from "./popover";
 import { Kbd } from "./kbd";
 
-const panelWidth = "400px";
-const itemHeight = "32px";
+const panelWidth = 400;
+const itemHeight = 32;
```

Comment:
please let's not rely on stitches magic where it is not colocated

Reply:
hmm ok